src/core/DialogBackground.qml

```diff
@@ -79,7 +79,7 @@ Rectangle {
                 GCText {
                     id: title
                     text: dialogBackground.title
-                    width: dialogBackground.width - 30
+                    width: dialogBackground.width - (30 + 60 * ApplicationInfo.ratio)
```

Comment:
it's a bit more complicated than this to fix.
When you resize the screen, you can see that the left anchor is the left part of the rectangle. It would be better for it to be the right size of the star difficulty level (else it overrides with it too).
Also, to what corresponds the 60 * ApplicationInfo.ratio?

Reply:
60 * Application.ratio corresponds to the width of the GCButtonCancel (cancel button).
So what we want here is to make the width of the text to be in between the difficulty stars and cancel button. is that correct ?

Reply:
is it possible to use directly button.width?
Yes, for the width of the text

Reply:
putting the difficulty icon and the title text in row with padding worked.
```
Row {
                    spacing: 2
                    padding: 4
                    Image {
                        id: titleIcon
                        /*
                        anchors {
                            left: parent.left
                            top: parent.top
                            margins: 4 * ApplicationInfo.ratio
                        }
                        */
                    }

                    GCText {
                        id: title
                        text: dialogBackground.title
                        width: dialogBackground.width - (30 + cancel.width)
                        horizontalAlignment: Text.AlignHCenter
                        verticalAlignment: Text.AlignVCenter
                        color: "black"
                        fontSize: 20
                        font.weight: Font.DemiBold
                        wrapMode: Text.WordWrap
                    }
                }
```